src/components/PersonList.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react';
+import { PersonComponent } from '../components/PersonComponent';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { Errors } from '../types/Errors';
+// type Props = {
+//   personList: Person[];
+// };
+
+export const PersonList: React.FC = () => {
+  const [personList, setPersonList] = useState<Person[]>([]);
+  const [isPersonListLoaded, setIsPersonListLoaded] = useState<boolean>(false);
+  const [personError, setPersonError] = useState<Errors | null>(null);
+  const getParent = (persons: Person[], personName: string | null) => {
+    return persons.find(pers => pers.name === personName);
+  };
+
+  const getErrorMessageBlock = () =>
+    personError === Errors.noPeopleMessage ? (
+      <p data-cy="noPeopleMessage">There are no people on the server</p>
+    ) : (
+      <p data-cy="peopleLoadingError" className="has-text-danger">
+        Something went wrong
+      </p>
+    );
+
+  useEffect(() => {
+    getPeople()
+      .then(persons => {
+        if (persons.length === 0) {
+          throw new Error(Errors.noPeopleMessage);
+        }
```

Comment:
You should not throw an error
You should display empty message if `persons.length === 0`